documentation/Using-OVS-DPDK-and-COR.md

```diff
@@ -0,0 +1,97 @@
+# Setup for Running OVS-DPDK
+
+The Data Plane Development Kit (DPDK) is a set of libraries and drivers for fast packet processing.  Open vSwitch (OVS) is an open-source implementation of a distributed multilayer switch.  Enabling DPDK support within OVS can yield significant performance improvements over linux-bridge, providing a switch with DPDK vhost-user ports.
+
+## Installation of DPDK and OVS on host system
+
+There is tight coupling between OVS and DPDK versioning, and what is available via distributions is often very out of date.  The directions below will instruct you how to build and configure from source. You MUST update the packages listed regularly to ensure your Clear Containers benefit from performance improvements, new features and most importantly CVE security fixes.
+
+As a prerequisite to building, you'll need a few packages on your host.  As an example, on an Ubuntu host system you'd need to install as follows:
+```
+$ sudo apt-get -y install autoconf automake kernel-common \
+    libpcap-dev libtool python python-six
+```
+
+Next we will set up DPDK:
+```
+$ git clone http://dpdk.org/git/dpdk -b v16.11 $HOME/dpdk
+$ cd $HOME/dpdk
+$ export RTE_SDK=$(pwd)
+$ export RTE_TARGET=x86_64-native-linuxapp-gcc
+$ make -j install T=$RTE_TARGET DESTDIR=install EXTRA_CFLAGS='-g'
+$ sudo cp -f install/lib/lib* /lib64/
+$ export DPDK_BUILD_DIR=x86_64-native-linuxapp-gcc
+```
+Once this completes with out issue, build and configure OVS:
+```
+$ git clone https://github.com/openvswitch/ovs -b branch-2.6 $HOME/ovs
+$ cd $HOME/ovs
+$ ./boot.sh
+$ ./configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var \
+    --with-dpdk="$RTE_SDK/$DPDK_BUILD_DIR"  --disable-ssl --with-debug  CFLAGS='-g'
+$ sudo -E make install -j
+```
+
+## Running OVS-DPDK on host system
+Once installed without error, we'll need to start up the ovsdb-server and ovs-vswitchd. The Open_vSwitch configuration settings will vary depending on the hardware you are running on (ex: lcore-mask, socket-mem). An example configuration is shown below, which should be adjusted for your setup:
+
+```
+$ sudo mkdir -p /var/run/openvswitch
+$ sudo killall ovsdb-server ovs-vswitchd
+$ rm -f /var/run/openvswitch/vhost-user*
+$ sudo rm -f /etc/openvswitch/conf.db
+
+$ export DB_SOCK=/var/run/openvswitch/db.sock
+$ sudo -E ovsdb-tool create /etc/openvswitch/conf.db /usr/share/openvswitch/vswitch.ovsschema
+$ sudo -E ovsdb-server --remote=punix:$DB_SOCK --remote=db:Open_vSwitch,Open_vSwitch,manager_options --pidfile --detach
```

Comment:
This is a quite a long list of commands. It might be better to provide a script (that calls `sudo` where needed) to make this process faster and less error-prone. Then tell the user to run it via `script` so they have a log in case things go wrong:

```
./setup-ovs.sh
```